lib/imgproxy/url_adapters/shrine.rb

```diff
@@ -17,7 +17,9 @@ def applicable?(image)
       end
 
       def url(image)
-        image.url(host: @host)
+        opts = {}
+        opts = { host: @host } if @host
+        image.url(**opts) 
```

Comment:
No need to use double splat here. Afaik providing hash instead of splatting it works better.